packages/preview/starter-journal-article/0.2.0/README.md

```diff
@@ -0,0 +1,206 @@
+# A configurable starter template for journal articles
+
+This package provides a template for writing journal articles
+to organise authors, institutions, and information of corresponding authors.
+
+![](./thumbnail.png)
```

Comment:
I recommend not adding the thumbnail here as Typst Universe will show a click-to-enlarge thumbnail anyway.

Reply:
Thanks. This has been removed in cee5160deee5f229c2755c49dc4fa39cf62ae991 .